sonyapilib/device.py

```diff
@@ -272,6 +281,32 @@ def _parse_ircc(self):
             self.ircc_url, method=HttpMethod.GET, raise_errors=True)
 
         upnp_device = f"{URN_UPNP_DEVICE}device"
+
+        self.friendly_name = self._find_device_info(response.text, "friendlyName",
+                                                    upnp_device=upnp_device)
+        self.manufacturer = self._find_device_info(response.text, "manufacturer",
+                                                   upnp_device=upnp_device)
+        self.manufacturer_url = self._find_device_info(response.text, "manufacturerURL",
+                                                       upnp_device=upnp_device)
+        self.model_description = self._find_device_info(response.text, "modelDescription",
+                                                        upnp_device=upnp_device)
+        self.model_name = self._find_device_info(response.text, "modelName",
+                                                 upnp_device=upnp_device)
+        self.model_url = self._find_device_info(response.text, "modelURL",
+                                                upnp_device=upnp_device)
+
+        icons = find_in_xml(
+            response.text,
+            [upnp_device,
+             f"{URN_UPNP_DEVICE}iconList",
+             (f"{URN_UPNP_DEVICE}icon", True),
+             f"{URN_UPNP_DEVICE}url"])
+
+        if not hasattr(self, 'ircc_base'):
```

Comment:
why check if the attrbute exists when it's always set in init?

Reply:
load_from_json will not call __init__ so it will not exist when loading old json files

Reply:
Yeah, it didn't like the old config when testing :D

I will add those tests in a few minutes.